Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,13 @@
 import React from 'react';
 import './Header.css';
+
+function HeaderTitle(props) {
+    return (
+      <div className="headerTitle">
+        <h3 className="title">{props.title}</h3>
```

Comment:
You could've used [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) here to make this slightly less to type:
```javascript
function HeaderTitle({ title }) {
  return (
    <div>
       <h3>{title}</h3>
       // ...
  );
}
```